layout.css

```diff
@@ -0,0 +1,7 @@
+.markdown-body {
+  max-width: 46em;
+  margin: 2em auto;
+  padding: 0 1em;
+  overflow: hidden;
+  word-wrap: break-word;
+}
```

Comment:
@bcomnes would be better to inline this in layout.html so there's no dependency on a module file path.

Reply:
I'll fix that

Reply:
Fixed in https://github.com/hypermodules/sitedown/commit/a5a5d16ec295af8dede53d90368e918da4a18dd3